relays/bin-substrate/src/cli/send_message.rs

```diff
@@ -60,12 +60,8 @@ pub struct SendMessage {
 	source: SourceConnectionParams,
 	#[structopt(flatten)]
 	source_sign: SourceSigningParams,
-	/// The SURI of secret key to use when transactions are submitted to the Target node.
-	#[structopt(long, required_if("origin", "Target"))]
-	target_signer: Option<String>,
-	/// The password for the SURI of secret key to use when transactions are submitted to the Target node.
-	#[structopt(long)]
-	target_signer_password: Option<String>,
+	#[structopt(flatten)]
+	target_sign: TargetSigningParams,
```

Comment:
This change means that now this particular cli option is checked during runtime. It isn't super-intuitive, but I haven't found a way to reuse existing struct && I do not want to create another one